### ISSUE TITLE
Fake screens

### ISSUE THREAD

**Issue by ramnes** [qtile/qtile#1238]
[This part of the documentation](https://github.com/qtile/qtile/blob/ed7198a5fb5438110f81a8c6ecc0e6289676c057/libqtile/config.py#L231-L232) mentions "fake screens", and the term is also found [in the code](https://github.com/qtile/qtile/blob/7c2a88fba68bdcf6f25dfb5494a74afc475d674e/libqtile/manager.py#L357-L373).

What are they? How to use them?

We need to document answers to those questions, and then make sure they work correctly.

See #1192 for this last point.

**Comment by ramnes** [comment]
@mattharrison @flacjacket can you help on this?

**Comment by kes31** [comment]
In my opinion they are very helphul, if one usese a 4k-Screen and wants to split it up to multiple fake screens.

**Comment by Melkor333** [comment]
@kes31 do you still know how it would work and could you give a simple example on how to configure fakescreens (assuming they work)?
I can probably spend some time to configure them for my widescreen monitor next week and create a documentation, if noone else is looking into it at moment.

**Comment by kes31** [comment]
@Melkor333 What I remember the following points:
- My problem was, that I could not overwrite the screen-setting which was detected from x11. Basically all my manual settings to screens were ignored.
- I found in code theses fakescreens, which basically overwrite the x11 setup. They were prepared to work, except this small patch in code.
- When I configured in my config the fakescreens variable, the x11 screen setup was ignored and my configured screens were used instead. In this way it was possible to create two screens on one  monitor.

Solemnly I decided to switch to awesomwm so I do not have an example anymore.
